src/components/NcAppNavigationItem/NcAppNavigationItem.vue

```diff
@@ -296,6 +309,15 @@ export default {
 			default: '',
 		},
 
+		/**
+		 * A direct link. This will be used as the `href` attribute.
+		 * This will ignore any `to` prop being defined.
+		 */
+		href: {
+			type: String,
+			default: '#',
```

Comment:
```suggestion
			default: null,
```
Otherwise the later `if (this.to && !this.href)` clause won't work as expected.

Reply:
Forgot to change it! Well caught :wink: